src/actions/articleActions.js

```diff
@@ -113,3 +115,28 @@ export const likeDislike = (payload, slug) => dispatch => {
       return toast.error(error.response.data.detail, { autoClose: false, hideProgressBar: true });
     });
 };
+
+export const updateArticle = (slug, newData) => dispatch => {
+  toast.dismiss();
+  dispatch(editArticleInititated(true));
+  axiosInstance
```

Comment:
Add a return statement. `updateArticle` should return a promise.

Reply:
Return statement has been added to return the promise